docs/network-policy.md

```diff
@@ -41,10 +41,55 @@ it, which determines its relative order among all tiers.
 **Note**: K8s NetworkPolicies will be enforced once all tiers have been
 enforced.
 
+### Tier CRDs
+
+Creating Tiers as CRDs allows users the flexibility to create and delete
+Tiers as per their preference i.e. not be bound to 5 static tiering options
+as was the case initially.
+
+An example Tier might look like this:
+```yaml
+apiVersion: security.antrea.tanzu.vmware.com/v1alpha1
+kind: Tier
+metadata:
+  name: mytier
+spec:
+  priority: 10
+  description: "my custom tier"
+```
+
+Tiers have the following characteristics:
+- Policies can associate themselves with an existing Tier by setting the `tier`
+  field in a Antrea NetworkPolicy CRD spec to the Tier's name.
+- A Tier must exist before an Antrea policy can reference it.
+- Policies associated with higher ordered (low `priority` value) Tiers are
+  enforced first.
+- No two Tiers can be created with same priorities.
+- Updating the Tier's `priority` field is unsupported.
+- Deleting Tier with existing references from policies is not allowed.
+- Deleting system generated Tier resources is not allowed.
```

Comment:
"system-generated tier" is not defined anywhere, so you may clarify that you mean the 5 tiers described below